project-base/storefront/components/Basic/Image/ImageOverlay.tsx

```diff
@@ -0,0 +1,5 @@
+export const ImageOverlay: FC = () => {
+    return (
+        <div className="pointer-events-none absolute inset-0 h-full w-full rounded-xl bg-backgroundMost opacity-25 transition-opacity duration-300 group-hover:opacity-0" />
```

Comment:
It feels too grey for the image, have you tried different opacity values? I think I remember from the UI meeting Martin suggesting 15% opacity, or was it too bright?

Reply:
Let's discuss and choose the most visually appealing color.

Reply:
I found a way to achieve this without using an Overlay. I applied the Tailwind class mix-blend-multiply, and the result looks great.

Reply:
Hell yes, this looks amazing!